70.app-update-module/docs.md

```diff
@@ -7,7 +7,29 @@ shortcode-core:
 github: false
 ---
 
-## app-update-module (1.0.0)
+---
```

Comment:
Suspicious `---` -> there is already one above. Is this intentional?

Reply:
Good catch!